### DISCUSSION TITLE
The ability to turn of dark mode

### DISCUSSION THREAD

**Discussion by TheKnarf** [withastro/starlight#1258] [Feature Requests]
### What version of `starlight` are you using?

N/A

### What is your idea?

I'd like to be able to turn of dark-mode.

### Why is this feature necessary?

I have a bunch of diagrams in my documentation page and no way to automatically get versions of those diagrams that work with both light-mode and dark-mode. Doing so would be a lot of manual work in the current state.

It's really weird that there doesn't seem to be an option to disable theme switching.

### Do you have examples of this feature in other projects?

Docusaurus have this feature out of the box:

https://docusaurus.io/docs/api/themes/configuration

I really regret spending so much time setting up Starlight when its so much more buggy and missing features that Docusaurus have.

### Participation

- [ ] I am willing to submit a pull request for this feature.

**Comment by delucis** [comment]
Sorry I missed this @TheKnarf! Closing as a duplicate of #949.
